internal/server/Server.go

```diff
@@ -10,28 +10,24 @@ import (
 const LootBoxCount = BikerAgentCount * 2
 const MegaBikeCount = BikerAgentCount / 2
 
+type IBaseBikerServer baseserver.IServer[objects.IBaseBiker]
```

Comment:
I think this might have to compose the IGameState interface explicitly? Otherwise not sure you can be passing it to the agent's updateGameState as an IGameState in the GameLoop()

Reply:
Yeah Harry and I spoke about this last night. Technically this is fine, since we don't add any functionality to the interface that's not in the base package. I agree though, for posterity, an explicit composition is better. I think he was going to change it later